Config.py

```diff
@@ -5,11 +5,15 @@
 
 class Config:
     """The class storing all the config parameters."""
-    def __init__(self, **kwargs) -> None:
+    def __init__(self, **kwargs) -> None: 
         self.local_ais = {}
+        # initialize other parameters
+        for key, value in kwargs.items():
```

Comment:
Why did you move this first? Now some specified settings in kwargs will be overwritten by what is below right?

Reply:
I was thinking about something else that does not matter now, will revert